GFS_layer/GFS_typedefs.F90

```diff
@@ -3519,6 +3579,9 @@ subroutine control_print(Model)
       print *, ' zhour             : ', Model%zhour
       print *, ' kdt               : ', Model%kdt
       print *, ' jdat              : ', Model%jdat
+
+      print *, 'whether to override surface radiative fluxes seen by the land surface model'
```

Comment:
I don't think this line is really necessary. The (good!) variable name `override_surface_radiative_fluxes` is pretty self-explanatory, and there is already an awful lot of stdout from the physics.